src/services/__tests__/messagesService.test.ts

```diff
@@ -244,6 +283,24 @@ describe("MessageService#getMessage", () => {
     });
   });
 
+  it("returns a message with attachments from the API", async () => {
+    mockGetMessage.mockImplementation(() =>
+      t.success(validApiWithAttachmentsMessageResponse)
+    );
+
+    const service = new MessageService(api);
+
+    const res = await service.getMessage(mockedUser, aValidMessageId);
```

Comment:
this test should mock `messageModel.getMessage()` to return a message with some
 `prescription_data` and NOT mock `service.getMessage()` (otherwise we're testing nothing here)